core/frontend/src/GraphicalEditingScope.ts

```diff
@@ -0,0 +1,203 @@
+    this.range = range;
+  }
+}
+
+/** Represents a period of time within an [interactive editing]($docs/learning/InteractiveEditing.md) session during which the
+ * geometry of elements being displayed in one or more [[Viewport]]s is being modified. Outside of such a scope, whenever the
+ * geometry within a [GeometricModel]($backend) changes new [[Tile]]s must be generated to reflect those changes in a viewport.
+ * Regenerating entire tiles each time individual elements change can be time-consuming, which may introduce an unacceptable delay
+ * between making a modification and seeing its result on the screen.
+ *
+ * Within the context of a graphical editing scope, no new tiles are generated. Instead, the geometry for any deleted or modified elements
+ * is hidden in the tile graphics, and additional temporary graphics are displayed for any newly-inserted or modified elements. Only when the
+ * scope exits are new tiles produced.
+ *
+ * The application decides when to enter and exit a graphical editing scope. A single interactive editing session may involve any number of
+ * editing scopes. Typically, applications will enter a new editing scope (after first exiting a previous scope, if one exists):
+ *  - When switching from a non-graphical workflow to one that involves editing geometry; or
+ *  - When changing which geometric model is being edited; or
+ *  - After performing an operation that creates or modifies a "large" number (perhaps hundreds?) of elements.
+ *
+ * An application should typically exit any graphical editing scope before:
+ *  - Pulling changesets; or
+ *  - Switching from a graphical editing workflow to some non-graphical workflow.
+ *
+ * Graphical editing scopes are only supported for [[BriefcaseConnection]]s opened in read-write mode that contain version 1.0.11 or newer of the BisCore schema.
+ * @see [[BriefcaseConnection.enterEditingScope]] to create a scope for a briefcase.
+ * @see [[BriefcaseConnection.editingScope]] to obtain a briefcase's current scope.
+ * @see [[exit]] to terminate a scope.
+ * @beta
+ */
+export class GraphicalEditingScope extends BriefcaseNotificationHandler implements EditingScopeNotifications {
+  public get briefcaseChannelName() { return IpcAppChannel.EditingScope; }
+
+  /** Maps model Id to accumulated changes to geometric elements within the associated model. */
+  private readonly _geometryChanges = new Map<Id64String, ModelChanges>();
+  private _disposed = false;
+  private _cleanup?: RemoveFunction;
+
+  /** The connection to the iModel being edited. */
+  public readonly iModel: BriefcaseConnection;
+
+  /** Event raised when a new scope is created for any [[BriefcaseConnection]].
+   * @see [[onExiting]] and [[onExited]] for complementary events.
+   */
+  public static readonly onEnter = new BeEvent<(scope: GraphicalEditingScope) => void>();
+
+  /** Event raised when this scope is about to exit.
+   * @see [[onBEnteregin]] for the complementary event.

```

Comment:
typo

Reply:
@williamkbentley do you understand why bmsBuild did not catch this?